### PR TITLE
Analysis/ output files

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Added output files view for analyses that don't have their own specialized page (Phylogenetic Tree, Sistr, Biohansel)

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1167,7 +1167,7 @@
 		<maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
 		<maven.liquibase.plugin.version>${liquibase.version}</maven.liquibase.plugin.version>
 		<maven.war.plugin.version>3.2.2</maven.war.plugin.version>
-		<maven.jetty.plugin.version>9.4.24.v20191120</maven.jetty.plugin.version>
```
Comment (deepsidhu85) [inline pom.xml]: Had accidentally pushed the local changes

Comment (joshsadam) [reply pom.xml]: Does this actually need to be updated though?

Comment (deepsidhu85) [reply pom.xml]: Yup good idea to update it. Updated in b08491fd725ca9fd305283ad0e87df84ab7c1cd4

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -2071,6 +2071,7 @@ AnalysisOutputs.jsonOutput=JSON Output
 AnalysisOutputs.textOutput=Text Output
 AnalysisOutputs.previewing=Previewing
 AnalysisOutputs.outputFilePreview=Output File Preview
+AnalysisOutputs.downloadAllFiles=Download All Files (.zip)
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/resources/i18n/messages.properties]: Fixed in b08491fd725ca9fd305283ad0e87df84ab7c1cd4

**Pull request merged**